docs/guides/save_tf_state_in_vkcs_s3.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: "vkcs"
+page_title: "Save Terraform state in VKCS S3"
+description: |-
+  Save Terraform state in VKCS S3
+---
+
+## Configure a bucket
+
+You will need a bucket to save the terraform state, you can do this by following one of these documentations: 
+1) [Create bucket using terraform](https://cloud.vk.com/docs/en/tools-for-using-services/terraform/how-to-guides/aws)
+2) [Create bucket using website or aws cli](https://cloud.vk.com/docs/en/storage/s3/service-management/buckets/create-bucket)
+
+~> **Note:** If you decide to create a bucket using terraform, it is important to create this bucket in another folder.
+
+## Configure terraform
+
+After that, in the main project, where we want to save terraform state in bucket, we need to add the following setting:
+```terraform
+terraform {
+  backend "s3" {
+    bucket     = "<bucket-name>"
+    key        = "<path/to/tfstate>"
+    region     = "<region>"
+    access_key = "<public access key>"
+    secret_key = "<secret key>"
+
+    skip_credentials_validation = true
+    skip_metadata_api_check     = true
+    skip_requesting_account_id  = true
+    skip_region_validation      = true
+    skip_s3_checksum            = true
+    endpoints = {
+      s3 = "<domain>"
+    }
+  }
+}
+```
+The region and domain must be set according to the [documentation](https://cloud.vk.com/docs/en/tools-for-using-services/terraform/how-to-guides/aws).
```

Comment:
here it is
lets place it above

Reply:
place it like note. If place it above, it will look strange

Reply:
Strange? At the first step a user should collect prerequisites: credentials, bucket, etc.
Next he creates a file by our template and fill the template with his values. What is a reason to split the values to two parts?

Reply:
Another options is to place the template at the first part. And next describe its parameters and specify ways to collect their values.

Reply:
I tried to rewrite it, please look again